py3status/module.py

```diff
@@ -311,7 +311,7 @@ def set_module_options(self, module):
         https://i3wm.org/i3status/manpage.html#_universal_module_options
         """
         self.i3s_module_options = {}
-        self.py3_module_options = {}
+        self.py3_module_options = {'block': {}}
```

Comment:
this is ugly, I think we can do better

Reply:
Used to be `self.block_option`. It's not going to be used for anything else other than `background`, `border` + its widths. Happy to hear your suggestion.

Reply:
is there anything special about the block stuff?  If not why not just all live happily together?

Reply:
Also, if users didn't specify anything, the block is empty so I'm not wasting time looping through nothing. Can be either `background` only, `border` stuffs only, or all of them.

Reply:
whatever this needs a reworking at some point to make it more efficient/readable so I can live with it but make a comment explaining why this is in the code so the reasoning isn't lost

Reply:
Left a comment.

Reply:
I have separated them to make it more readable. Three dict options. `py3status` dict will not have more than two options max.